test/source/mock/attester/attester-endpoints.ts

```diff
@@ -435,3 +302,99 @@ NaqFqwp9jMCYEnjfM2VSYFTKFrQYm9zYUTIKnQxNtWC8Z85UKz2eMk/INukzWe7g
 cvQabqad/ZghLSTzo/Kf
 =sshZ
 -----END PGP PUBLIC KEY BLOCK-----`;
+
+interface PubKeyLookUpResult {
+  pubkey?: string;
+  returnError?: { code: number; message: string };
+}
+
+export interface AttesterConfig {
+  pubkeyLookup?: Record<string, PubKeyLookUpResult>;
+  ldapRelay?: Record<string, PubKeyLookUpResult>;
+  welcomeMessageEnabled?: boolean;
+}
+
+const LEGACY_GLOBAL_ATTESTER_MOCK_CONFIG: AttesterConfig = {
+  pubkeyLookup: {
+    'ci.tests.gmail@flowcrypt.test': {
+      pubkey: somePubkey,
+    },
+    'test.flowcrypt.pubkey.timeout@gmail.com': {
+      returnError: {
```

Comment:
@sosnovsky, to avoid further rewriting work, I would like to confirm with you regarding the AttesterConfig architecture.
I intend to use this architecture because we need to return the public key result for certain emails and return an error (like a timeout) for other emails. 
What do you think?
Let me know if you have better architecture in your mind.

```ts
pubkeyLookup?: Record<string, PubKeyLookUpResult>
interface PubKeyLookUpResult {
  pubkey?: string;
  returnError?: { code: number; message: string };
}

```

Instead of using this
```ts
pubkeyLookup?: Record<string, string>
returnError?: { code: number; message: string };

```

Reply:
Such architecture looks good 👍

Reply:
Sounds good!. Do you agree processing with above architecture?
Any other suggestions?

Reply:
Yes, let's use this one, looks simple and customizable for different test cases